content/tr/cloud-native-tech.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Cloud Native Teknolojisi
+status: Completed
+category: Kavram
+tags: ["Fundamental", "", ""]
+---
+
+## Nedir
+
+Cloud native yığını olarak da adlandırılan cloud native teknolojileri, cloud native uygulamaları oluşturmak için kullanılan teknolojilerdir.
+Bu teknolojiler kuruluşlar için genel, özel ve hibrit bulut ortamları gibi modern ve dinamik ortamlarda ölçeklenebilir uygulamalar oluşturmaya ve çalıştırmaya olanak tanırken [bulut bilişimin](/tr/cloud-computing/) yararlarını en üst düzeye çıkarırlar.
+Bulut bilişimin yeteneklerinden yararlanmak için sıfırdan tasarlanmışlardır ve konteynerler, servis ağları, mikro servisler ve sabit altyapı bu yaklaşımın örnekleridir.
+
+## Hangi sorunu çözer
+
+Cloud native yığını, zorlukların çeşidini adresleyen birçok farklı teknoloji kategorisine sahiptir.
+[CNCF Cloud Native Landscape’e](https://landscape.cncf.io/) baktığınızda ne kadar farklı alana temas ettiğini görebilirsiniz.
+Ancak genel olarak bahsedecek olursak cloud native teknolojileri, bir ana zorluk kümesini ele almaktadır; geleneksel BT işletim modellerinin olumsuz yönleri.
+Bu zorluk kümesi, ölçeklenebilir, arızalanırsa bile çalışmaya devam edebilen, kendi kendini onaran uygulamalar oluşturma güçlüklerinin yanı sıra kaynakların verimsiz kullanımını içermektedir.
+
+## Sorunu nasıl çözer
```

Comment:
```suggestion
## Nasıl Yardımcı Olur
```

We used this header in https://deploy-preview-2344--cncfglossary.netlify.app/tr/cloud-computing/.

Let's use the same for consistency.